jsonrpc/manager.py

```diff
@@ -96,8 +98,10 @@ def _get_responses(cls, requests, dispatcher):
         """
         for request in requests:
             def response(**kwargs):
-                return cls.RESPONSE_CLASS_MAP[request.JSONRPC_VERSION](
+                resp = cls.RESPONSE_CLASS_MAP[request.JSONRPC_VERSION](
```

Comment:
May you call `resp` as `response` please? This is absolutely minor issue but it ensures the consistency.

Reply:
fixed